From ec1ca18ac69c951308ad580da11fa4ed684894c9 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Sun, 9 Apr 2023 15:15:47 +0200
Subject: [PATCH] CMakeLists.txt: Fix hardcoded 'lib' installation path

:Release Notes:
With multilib enabled the libdir is usr/lib64 instead of usr/lib.

:Detailed Notes:
Respect WEBOS_INSTALL_LIBDIR to fix:
http://gecko.lge.com:8000/Errors/Details/574288

ERROR: QA Issue: loc-utils: Files/directories were installed but not shipped in any package:
  /usr/lib/libloc_utils.so.1
  /usr/lib/libloc_utils.so.1.0.0
  /usr/lib/libloc_utils.so
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
loc-utils: 3 installed and not shipped files. [installed-vs-shipped]

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRP-12446] CCC: Various build fixes
[WRP-12469] Bitbake world issues 2023-04-01

Change-Id: I8a15529a12eb69071e55ab78ba3cca113779d077
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/com.webos.service.location/+/350819 CMakeLists.txt: Fix hardcoded 'lib' installation path]

 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d47d739..c9d71ae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,7 +72,7 @@ include_directories("${PROJECT_SOURCE_DIR}/include")
 
 
 set(DB_UTILS_NAME dbutils)
-set(DB_UTILS_LIB_PATH ${CMAKE_INSTALL_PREFIX}/lib)
+set(DB_UTILS_LIB_PATH ${WEBOS_INSTALL_LIBDIR})
 set(DB_UTILS_DATA_PATH ${WEBOS_INSTALL_LOCALSTATEDIR}/location)
 set(DB_UTILS_SRC ${PROJECT_SOURCE_DIR}/src/utils/db_util.cpp ${PROJECT_SOURCE_DIR}/src/utils/Gps_stored_data.cpp)
 set(LIBRARIES ${GLIB2_LDFLAGS} ${LIB2XML_LDFLAGS})
